apps/prairielearn/src/pages/partials/questionFooter.ejs

```diff
@@ -74,76 +86,6 @@
-      <div class="row">
-        <div class="col d-flex justify-content-between">
-          <span class="d-flex align-items-center">
-              <% if (showSaveButton) { %>
-                <button class="btn btn-info question-save disable-on-submit order-2">
-                  <% if (showGradeButton) { %>
-                    Save only
-                  <% } else { %>
-                    Save
-                  <% } %>
-                </button>
-              <% } %>
-              <% if (showGradeButton) { %>
-                <button class="btn btn-primary question-grade disable-on-submit order-1 mr-1" <%= disableGradeButton ? "disabled" : "" %> >Save & Grade
-            <% if (variantAttemptsTotal > 0) { %>
-               <% if (variantAttemptsLeft > 1) { %>
-               <small class="font-italic ml-2"><%= variantAttemptsLeft %> attempts left</small>
-               <% } else if (variantAttemptsLeft == 1 && variantAttemptsTotal > 1) { %>
-               <small class="font-italic ml-2">Last attempt</small>
-               <% } else if (variantAttemptsLeft == 1) { %>
-               <small class="font-italic ml-2">Single attempt</small>
-               <% } %>
-            <% } else if (question_context != 'instructor') { %>
-               <small class="font-italic ml-2">Unlimited attempts</small>
-            <% } %>
-            </button>
-              <% } %>
-          </span>
-          <span class="d-flex">
-            <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
-            <input type="hidden" name="postData" class="postData">
-            <input type="hidden" name="__action" class="__action">
-            <% if (showNewVariantButton) { %><a href="<%= newVariantUrl %>" class="btn btn-primary disable-on-click ml-1">New variant</a>
-            <% } else if (showTryAgainButton) { %><a href="<%= tryAgainUrl %>" class="btn btn-primary disable-on-click ml-1">Try a new variant</a>
-            <% } else if (hasAttemptsOtherVariants) { %>
-            <small class="font-italic align-self-center">Additional attempts available with new variants</small>
-            <a class="btn btn-xs" data-toggle="popover" data-html="true" data-content="<%= include('newVariantInfo') %>" data-placement="auto" data-trigger="focus" data-container="body" tabindex="0">
-              <i class="fa fa-question-circle" aria-hidden="true"></i>
-            </a>
-            <% } %>
-            <% if (question_context == 'student_exam' && instance_question.points_list) { %>
-            <% const rounded_points = instance_question.points_list.map((p) => Math.round(p * 100) / 100); %>
-            <small class="font-italic align-self-center text-right">
-              <% if (rounded_points[0] == 1) { %>
-              1 point available for this attempt
-              <% } else { %>
-              <%= rounded_points[0] %> points available for this attempt
-              <% } %>
-              <% if (rounded_points.length == 2) { %>
```

Comment:
At this point Freeform includes a note regarding the split of auto and manual grading, and v2 did not. I kept the freeform behaviour for both.